desktop_version/src/Entity.cpp

```diff
@@ -1,6 +1,7 @@
 #define OBJ_DEFINITION
 #include "Entity.h"
 
+#include <cmath>
```

Comment:
Why `#include` this? I don't see any functions from it being used here. (And this is a moot point, but if you were to `#include` it, please do `#include <math.h>` instead of `#include <cmath>`.)